app/src/main/java/cto/its/ny/gov/fakenews/MainActivity.kt

```diff
@@ -23,110 +24,36 @@ class MainActivity : AppCompatActivity() {
         setContentView(view)
 
         val amountOfArticles = 0..5
-        createArticles(amountOfArticles)
+        val nSvc = NewsService()
+        arrayOfArticles = nSvc.createArticles(amountOfArticles)
```

Comment:
Awesome job making the `NewsService.createArticles()` method flexible with an `IntRange` parameter. 🎉